src/persist-client/src/lib.rs

```diff
@@ -102,7 +102,7 @@ impl PersistLocation {
         let blob = BlobConfig::try_from(&self.blob_uri).await?;
         let blob =
             retry_external(&metrics.retries.external.blob_open, || blob.clone().open()).await;
-        let consensus = ConsensusConfig::try_from(&self.consensus_uri).await?;
+        let consensus = ConsensusConfig::try_from(&self.consensus_uri, 1).await?;
```

Comment:
let's pass in a `&PersistConfig` to this method so we can also use consensus_connection_pool_max_size here